.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "avalanche/lib/openzeppelin-contracts-upgradeable"]
 	path = avalanche/lib/openzeppelin-contracts-upgradeable
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable
+[submodule "ethereum/lib/forge-std"]
+	path = ethereum/lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
```

Comment:
You shouldn't need an extra submodule for this on the eth side. You should be able to set `remappings.txt` to resolve to whatever you need it to in order to use the already existing submodule.